ParcelDistributionCenter/ParcelDistributionCenter.Model/Models/AssignPackagesVM.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Newtonsoft.Json;
+using ParcelDistributionCenter.Model.Enums;
+using ParcelDistributionCenter.Model.Models;
+using System.ComponentModel.DataAnnotations;
+using System.Xml.Linq;
+
+namespace ParcelDistributionCenter.Web.Models
+{
+
+    public class AssignPackagesVM
```

Comment:
Tu robisz nowy VM? To wydaje mi się, że to powinno być W projekcie Web. Ponadto zastanowiłbym się, czy nie da się jakoś tych samych właściwości zamiast powielać, jakoś tutaj wpiać z innego miejsca, żeby nei powielać kodu

Reply:
Masz racje. Do przeniesienia do Web.

Co do powielania to nie do końca mam pomysł jak to zrobić.